infra/terraform/eks/main.tf

```diff
@@ -0,0 +1,49 @@
+provider "azurerm" {
```

Comment:
azure? Some string replacement is still needed...

Reply:
yes, I pushed to the wrong origin